presto-main/src/test/java/com/facebook/presto/sql/planner/assertions/WindowMatcher.java

```diff
@@ -46,24 +46,23 @@ public boolean matches(PlanNode node, Session session, Metadata metadata, Symbol
         }
 
         WindowNode windowNode = (WindowNode) node;
-        Collection<FunctionCall> actualCalls = windowNode.getWindowFunctions().values();
+        List<FunctionCall> actualCalls = windowNode.getWindowFunctions().values().stream().map(function -> function.getFunctionCall()).collect(toList());
 
         if (actualCalls.size() != functionCalls.size()) {
             return false;
         }
 
         LinkedList<FunctionCall> expectedCalls = new LinkedList<>(functionCalls);
```

Comment:
My fault. I was thinking of `actualCopy`.
Actually this is probably a mistake here then.
I think Eric wanted `actualCopy` to be a `LinkedList`, not `expectedCall`. `Collectors.toList()` does not guarantee what kind of `List` is returned.
Can you confirm, @ebd2?

I think it's right to patch this. Here's my proposal: b545c5c7a711a6432f98bafff8736dded01acfcb